improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -1208,10 +1208,9 @@ def _recycle_raw_ensemble_realizations(
                 raw_forecast_realization = raw_forecast_realizations.extract(constr)
                 raw_forecast_realization.coord("realization").points = index
                 raw_forecast_realizations_extended.append(raw_forecast_realization)
-            raw_forecast_realizations = concatenate_cubes(
-                raw_forecast_realizations_extended,
-                coords_to_slice_over=["realization", "time"],
-            )
+            raw_forecast_realizations = ConcatenateCubes(
+                coords_to_slice_over=["realization", "time"]
+            )(raw_forecast_realizations_extended)
```

Comment:
Good idea.  I think this is served by `MergeCubes`, potentially with slicing over realizations?  I'm not familiar enough with ECC to fully understand the use case yet, but since I'm currentky refactoring the unit tests I should understand it better after that work - at which point I can make the further changes needed to retire `ConcatenateCubes` properly.